src/pages/admin/reports/components/funds-expenditures-section/components/common/delete-funds-expenditures-category-modal/DeleteFundsExpendituresCategoryModal.module.scss

```
@@ -81,10 +85,21 @@
 }
 
+.labelRow {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 8px;
+}
+
 .label {
     @include type.small-text-medium;
     color: c.$bluish-black;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)